### ISSUE TITLE
Docker

### ISSUE THREAD

**Issue by nightlyworker** [h3rald/litestore#58]
Hi guys,
Very elegant and beautiful project. Thank you for open sourcing it. I apologize for creating issue instead of pull request (yay time!).

I did like to share with you following Docker file which you can expand/adapt/publish official images to drive more adoption across community.

Dockerfile
```dockerfile
FROM alpine:3.12

ARG USER=default
ENV HOME /home/$USER

RUN apk add --no-cache gcc musl-dev git sudo

# add new user
RUN adduser -D $USER && \
		echo "$USER ALL=(ALL) NOPASSWD: ALL" > /etc/sudoers.d/$USER && \
		chmod 0440 /etc/sudoers.d/$USER

# install nim and nimble (# https://github.com/Docker-Hub-frolvlad/docker-alpine-nim/blob/master/Dockerfile)
RUN export NIM_VERSION=1.4.0 && \
    export NIMBLE_VERSION=0.12.0 && \
    \
    apk add --no-cache libcrypto1.1 libssl1.1 && \
    apk add --no-cache --virtual=.build-dependencies wget ca-certificates git && \
    mkdir -p "/opt" && \
    \
    cd "/opt" && \
    wget "https://github.com/nim-lang/Nim/archive/v$NIM_VERSION.tar.gz" -O - | tar xz && \
    mv "./Nim-$NIM_VERSION" "./Nim" && \
    cd "./Nim" && \
    wget "https://github.com/nim-lang/csources/archive/master.tar.gz" -O - | tar xz && \
    mv "./csources-master" "./csources" && \
    cd "./csources" && \
    sh build.sh && \
    cd .. && \
    ./bin/nim c koch && \
    ./koch boot -d:release && \
    chmod +x "/opt/Nim/bin/nim" && \
    ln -s "/opt/Nim/bin/nim" "/usr/local/bin/nim" && \
    rm -r "./csources" "./tests" && \
    \
    cd "/opt" && \
    wget "https://github.com/nim-lang/nimble/archive/v$NIMBLE_VERSION.tar.gz" -O - | tar xz && \
    cd "./nimble-$NIMBLE_VERSION" && \
    nim compile --run "src/nimble" build --accept && \
    rm -rf /tmp/* && \
    chmod +x nimble && \
    mv nimble "/usr/local/bin/" && \
    rm -rf "/opt/nimble-$NIMBLE_VERSION" && \
    \
    apk del .build-dependencies 

# install litestore and dependencies
RUN nimble install --verbose -y nimgen && \
    ln -s /home/default/.nimble/bin/nimgen /usr/bin/nimgen && \
   nimble install --verbose -y c2nim && \
   ln -s /home/default/.nimble/bin/c2nim /usr/bin/c2nim && \
  nimble install --verbose -y litestore && \
  ln -s /home/default/.nimble/bin/litestore /usr/bin/litestore
USER $USER
WORKDIR $HOME
ENTRYPOINT ["litestore"]
```		

Mini how-to

Paste content of Dockerfile and create Dockerfile anywhere.
Build the image
```bash
docker build -t martin/litestore:0.1 .
``` 
Use the image
```sh
docker run martin/litestore:0.1 -v
1.9.3
```

Start for normal usage inline with Litestore guidelines
```sh
docker run -p 9500:9500 martin/litestore:0.1 -a:0.0.0.0
curl http://localhost:9500/
{
  "version": "LiteStore v1.9.3",
  "datastore_version": 2,
  "api_version": 7,
  "size": "0.07 MB",
  "read_only": false,
  "log_level": "warn",
  "directory": null,
  "mount": false,
  "additional_stores": [],
  "auth": false,
  "total_documents": 0,
  "total_tags": 0,
  "tags": []
}%               
```

Thank you,
Martin

**Comment by h3rald** [comment]
Awesome, thanks a lot!!! I'll make the changes for next release and add the nice howto you made to the user guide 👍

**Comment by h3rald** [comment]
Documented in v1.10.0, official images available (h3rald/LiteStore:v1.10.0).
